endorsement/manager.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2021 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package endorsement
+
+import (
+	"github.com/hashicorp/go-plugin"
+
+	"veraison/common"
+)
+
```

Comment:
A one liner comment about manager might be helpful here.

Reply:
Ah, missed that. Now added. Thanks!